tests/VersionRouteTest.php

```diff
@@ -0,0 +1,248 @@
+    protected const ROUTE_FACADE_TO = '/test-facade-to';
+    protected const ROUTE_OBJECT_RANGE = '/test-object-range';
+    protected const ROUTE_OBJECT_FROM = '/test-object-from';
+    protected const ROUTE_OBJECT_TO = '/test-object-to';
+
+    protected const ROUTE_FACADE_VERSION = '/test-facade-version';
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->app->bind(VersionEnumContract::class, VersionEnum::class);
+    }
+
+    public function getTestVersionRangeData(): array
+    {
+        return [
+            // Range
+            [
+                'version' => '1',
+                'is_correct_version' => true,
+                'route' => static::ROUTE_OBJECT_RANGE,
+            ],
+            [
+                'version' => '1.0',
+                'is_correct_version' => false,
+                'route' => static::ROUTE_OBJECT_RANGE,
+            ],
+            [
+                'version' => '0.5',
+                'is_correct_version' => false,
+                'route' => static::ROUTE_OBJECT_RANGE,
+            ],
+            [
+                'version' => '4',
+                'is_correct_version' => false,
+                'route' => static::ROUTE_OBJECT_RANGE,
+            ],
+            [
+                'version' => '2',
+                'is_correct_version' => true,
+                'route' => static::ROUTE_OBJECT_RANGE,
+            ],
+            [
+                'version' => '2.0',
+                'is_correct_version' => false,
+                'route' => static::ROUTE_OBJECT_RANGE,
+            ],
+            [
+                'version' => '1.5',
+                'is_correct_version' => false,
+                'route' => static::ROUTE_OBJECT_RANGE,
+            ],
+
+            [
+                'version' => '1',
+                'is_correct_version' => true,
+                'route' => static::ROUTE_FACADE_RANGE,
+            ],
+            [
+                'version' => '1.0',
+                'is_correct_version' => false,
+                'route' => static::ROUTE_FACADE_RANGE,
+            ],
+            [
+                'version' => '0.5',
+                'is_correct_version' => false,
+                'route' => static::ROUTE_FACADE_RANGE,
+            ],
+            [
+                'version' => '4',
+                'is_correct_version' => false,
+                'route' => static::ROUTE_FACADE_RANGE,
+            ],
+            [
+                'version' => '2',
+                'is_correct_version' => true,
+                'route' => static::ROUTE_FACADE_RANGE,
+            ],
+            [
+                'version' => '2.0',
+                'is_correct_version' => false,
+                'route' => static::ROUTE_FACADE_RANGE,
+            ],
+            [
+                'version' => '1.5',
+                'is_correct_version' => false,
+                'route' => static::ROUTE_FACADE_RANGE,
+            ],
+
+            // From
```

Comment:
please implement several tests instead of comments

Reply:
Done.